contrib/kind.sh

```diff
@@ -481,12 +483,12 @@ coredns_patch() {
 build_ovn_image() {
   if [ "$OVN_IMAGE" == local ]; then
     # Build ovn docker image
-    pushd ../go-controller
+    pushd ${DIR}/../go-controller
     make
     popd
 
     # Build ovn kube image
-    pushd ../dist/images
+    pushd ${DIR}/../dist/images
     # Find all built executables, but ignore the 'windows' directory if it exists
     find ../../go-controller/_output/go/bin/ -maxdepth 1 -type f -exec cp -f {} . \;
```

Comment:
do you want ${DIR} here too?

Reply:
right, so you want it to be:

`find ${DIR}/../../go-controller/_output/go/bin/ -maxdepth 1 -type f -exec cp -f {} . \;`

right?